From 3b19c8cd2da7060574304c98d79f23be3158f297 Mon Sep 17 00:00:00 2001
From: Antonio Ospite <antonio.ospite@collabora.com>
Date: Mon, 20 Jan 2025 09:38:46 +0100
Subject: [PATCH] Fix building on Android with minimal dependencies
X-Face: z*RaLf`X<@C75u6Ig9}{oW$H;1_\2t5)({*|jhM<pyWR#k60!#=#>/Vb;]yA5<GWI5`6u&+
 ;6b'@y|8w"wB;4/e!7wYYrcqdJFY,~%Gk_4]cq$Ei/7<j&N3ah(m`ku?pX.&+~:_/wC~dwn^)MizBG
 !pE^+iDQQ1yC6^,)YDKkxDd!T>\I~93>J<_`<4)A{':UrE

The fixes the following error:

+ gn gen out/Android
ERROR at //build/android/BUILD.gn:233:5: Unable to load "/angle-android-build/third_party/catapult/tracing/BUILD.gn".
    "//third_party/catapult/tracing:convert_chart_json",
    ^--------------------------------------------------
---
 android/BUILD.gn | 1 -
 1 file changed, 1 deletion(-)

diff --git a/android/BUILD.gn b/android/BUILD.gn
index 3c6fc6ab5..db25974a6 100644
--- a/android/BUILD.gn
+++ b/android/BUILD.gn
@@ -230,7 +230,6 @@ python_library("resource_sizes_py") {
   pydeps_file = "resource_sizes.pydeps"
   data_deps = [
     ":devil_chromium_py",
-    "//third_party/catapult/tracing:convert_chart_json",
   ]
 
   data = [
-- 
2.47.1

